README.md

```diff
@@ -13,39 +13,119 @@ Artifacts in support of ODH Edge use cases that show integration with Red Hat Ad
+and deploy it at the near edge location(s) (represented by ACM-managed clusters) in a centralized way.
+
+### Developing and training a model
 
-### Infrastructure Configuration
+This step is out of scope of this Proof of Concept repository,
+as this repository already contains trained models in the [pipelines/models/](pipelines/models/) directory.
 
-1. Provision OpenShift Cluster
-1. Configure the default Identity Provider
-1. Install Red Hat Advanced Cluster Management
-1. Register the clusters
+If you wish to develop and train different models,
+Jupyter notebooks provided by [Open Data Hub](https://opendatahub.io/) (ODH)
+or [Red Hat OpenShift Data Science](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science) (RHODS) can be used.
+To install ODH or RHODS operators, admin privileges in the OpenShift cluster are needed.
 
-   [ACM Application](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/applications/managing-applications) manifests are located in [acm/registration](acm/registration) to register and configure the target environments required for the AI at the Edge use cases.  The files can be applied to the ACM hub cluster manually:
-   ```
-   $  oc apply -k acm/registration/
-   ```
-   * Core - Cluster host the ODH Core components that will be used in the MLOps Engineer workflow to train, build and push the model.  This cluster is not required to be co-located with the ACM Hub but we group them together to simplify the use case
-   * Near Edge - Cluster(s) that will host the running model at the edge.  This is the target environment after a new model is available for use
-1. Deploy Open Data Hub to the Core cluster and register any configurations to support pushing models to the edge cluster
-   * Open Data Hub - v2.x of the operator will be installed as part of the dependencies for the core cluster.  Manual installation of the [DataScienceCluster](https://github.com/opendatahub-io/opendatahub-operator#example-datasciencecluster) object will be required to deploy the Open Data Hub core stack to support the data science workflow
-   * GitOps repos
-   * Image repos
-1. Manage the edge cluster environments to support deployment of models and support for monitoring
-   * Configure ACM Observability
-   * Deploy the Model container
-
-### MLOps Engineer workflows
-
-1. Develop the model in an ODH Jupyter notebook
-1. Build the model from the notebook using Data Science Pipelines
-1. Push the model to the image registry accessible by the near edge cluster(s)
-1. Update the GitOps config for the near edge cluster
-
-### Pipelines setup
-
-See [pipelines/README.md](pipelines/README.md)
+Working and deploying your own models might require bigger changes
+to the definition and configuration of the pipelines and ACM setup below,
+so you might want to start with the pre-built models first.
+
+### Building container image with model and serving runtime
```

Comment:
```suggestion
### Building container image(s) with a model and serving runtime
```

Reply:
Applied in 49aadf7f7022a32884be3d91d13200f348ad82ab.